cadasta/gui/tools/wizard/step_project_update02.py

```diff
@@ -0,0 +1,210 @@
+
+        self.update_status_label.setText('')
+
+        self.project_name_text.setText(self.project['name'])
+
+        if self.project['urls']:
+            self.project_url_text.setText(self.project['urls'][0])
+        else:
+            self.project_url_text.setText('')
+
+        self.project_desc_text.setText(self.project['description'])
+
+        if self.project['access'] == 'private':
+            private_access = True
+        else:
+            private_access = False
+        self.access_checkbox.setChecked(private_access)
+
+        contacts = Contact.get_rows()
+        project_contacts = self.project['contacts']
+
+        self.project_contact_list.clear()
+        self.project_contact_list.setSelectionMode(
+            QAbstractItemView.ExtendedSelection
+        )
+
+        # From local db
+        for index, contact in enumerate(contacts):
+            contact_name = contact.name
+            contact_email = ' - ' + contact.email if contact.email else ''
+            contact_phone = ' - ' + contact.phone if contact.phone else ''
+
+            contact_item = QListWidgetItem(
+                contact_name + contact_email + contact_phone
+            )
+
+            contact_item.setData(Qt.UserRole, contact)
+            self.project_contact_list.addItem(
+                contact_item
+            )
+
+        # From server
+        for index, contact in enumerate(project_contacts):
+
+            contact_name = contact['name']
+            contact_email = ' - ' + contact['email'] \
+                if contact['email'] else ''
+            contact_phone = ' - ' + contact['tel'] if contact['tel'] else ''
+
+            contact_box = self.project_contact_list.findItems(
+                contact_name + contact_email + contact_phone,
+                Qt.MatchExactly
+            )
+
+            if len(contact_box) > 0:
+                contact_box[0].setSelected(True)
+            else:
+                new_contact = Contact()
+                new_contact.name = contact['name']
+                new_contact.email = contact['email']
+                new_contact.phone = contact['tel']
+                new_contact.save()
+
+                selected_item = QListWidgetItem(
+                    contact_name + contact_email + contact_phone
+                )
+                selected_item.setData(Qt.UserRole, new_contact)
+
+                self.project_contact_list.addItem(
+                    selected_item
+                )
+
+                selected_item.setSelected(True)
+
+    def validate_step(self):
+        """Check if the step is valid.
+
+        :returns: Tuple of validation status and error message if any
+        :rtype: ( bool, str )
+        """
+        return (
+            True,
+            ''
+        )
+
+    def get_next_step(self):
+        """Find the proper step when user clicks the Next button.
+
+        :returns: The step to be switched to
+        :rtype: WizardStep instance or None
```

Comment:
:rtype: should be an actual python type (or list of types separated by commas e.g.

WizardStep, None